ubuntu/ubuntu-18.x/ubuntu-18.04-LTS-hpc/install_mpis.sh

```diff
@@ -20,20 +19,20 @@ $COMMON_DIR/download_and_verify.sh $HPCX_DOWNLOAD_URL "83f03e2d01cb1e4198e50ed9b
 tar -xvf ${TARBALL}
 mv ${HPCX_FOLDER} ${INSTALL_PREFIX}
 HPCX_PATH=${INSTALL_PREFIX}/${HPCX_FOLDER}
+$COMMON_DIR/write_component_version.sh "HPCX" ${HPCX_VERSION}
 
-# MVAPICH2 2.3.6
-MV2_VERSION="2.3.6"
-$COMMON_DIR/write_component_version.sh "MVAPICH2" ${MV2_VERSION}
+# MVAPICH2 2.3.7
+MV2_VERSION="2.3.7"
 MV2_DOWNLOAD_URL=http://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich2-${MV2_VERSION}.tar.gz
-$COMMON_DIR/download_and_verify.sh $MV2_DOWNLOAD_URL "b3a62f2a05407191b856485f99da05f5e769d6381cd63e2fcb83ee98fc46a249"
+$COMMON_DIR/download_and_verify.sh $MV2_DOWNLOAD_URL "c39a4492f4be50df6100785748ba2894e23ce450a94128181d516da5757751ae"
 tar -xvf mvapich2-${MV2_VERSION}.tar.gz
 cd mvapich2-${MV2_VERSION}
 ./configure --prefix=${INSTALL_PREFIX}/mvapich2-${MV2_VERSION} --enable-g=none --enable-fast=yes && make -j$(nproc) && make install
 cd ..
+$COMMON_DIR/write_component_version.sh "MVAPICH2" ${MV2_VERSION}
 
 # OpenMPI 4.1.1
```

Comment:
same comments as the other file (pls check versions)

Reply:
Updated